syntax = "proto3";
package tutorial;

option go_package = 'github.com/gabrielcipriano/distributed-systems-class/tree/main/project2-grpc-middleware/client-go';

message putRequest {
  optional string key = 1;
  optional int32 value = 2;
}

message putResponse {
  optional bool ok = 1;
}

message getRequest {
  optional string key = 1;
}

message getResponse {
  optional int32 value = 1;
}